docs/website/tds/tutorial/NcML.htm

```diff
@@ -335,10 +335,10 @@ <h2><a name="debug">Debugging NcML</a></h2>
         <li>Remove the <code>recheckEvery</code> attribute if present on the <code>&lt;scan&gt;</code> element.</li>
         <li>Make sure that the <code>&lt;scan&gt;</code> location is available on the machine you are running ToolsUI</li>
       </ol>
-      Now start up <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">ToolsUI</a>, and in the viewer tab, navigate to
+      Now start up <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">ToolsUI</a>, and in the viewer tab, navigate to
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.